docs/BUILD_PIPELINE_DEBUGGING.md

```diff
@@ -0,0 +1,30 @@
+# Build Pipeline Debugging
```

Comment:
Oho there is a various tips and tricks section which contains some debugging info. In that section, adding this might be helpful.